server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/RecipientRewriteTableTest.java

```diff
@@ -29,31 +41,13 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import javax.mail.Message;
-import javax.mail.MessagingException;
-import javax.mail.internet.MimeMessage;
-import java.util.ArrayList;
-import java.util.Iterator;
-import java.util.Properties;
-
-import static org.apache.james.transport.mailets.RecipientRewriteTableMock.mapFrom;
-import static org.apache.james.transport.mailets.RecipientRewriteTableMock.rewriteTableMock;
-import static org.junit.Assert.assertEquals;
-
 public class RecipientRewriteTableTest {
 
     private org.apache.james.transport.mailets.RecipientRewriteTable table;
 
     @Before
     public void setUp() throws Exception {
-        final FakeMailContext mockMailetContext = new FakeMailContext() {
-
-            @Override
-            public boolean isLocalServer(String serverName) {
-                return serverName.equals("localhost");
-
-            }
-        };
```

Comment:
No, it's part of this commit, as i limit the visibility of the FakeMailContext constructor in this one.